src/xdocs/checks/modifier/redundantmodifier.xml

```diff
@@ -153,6 +157,13 @@ public class ClassExtending extends ClassExample {
               <th>default value</th>
               <th>since</th>
             </tr>
+            <tr>
+              <td>jdkVersion</td>
+              <td>Set the JDK version that you are using. We support JDK versions in both notations for versions up to and including 8 (e.g., 5 and 1.5, 8 and 1.8). For versions 9 and later, use the single version number (e.g., 9, 10, 11, etc.).</td>
```

Comment:
I find it odd you say "both" versions before you identify what the different versions are. I would avoid pronouns as it is the documentation conveying to the user what is going on, not us directly.

`We support JDK versions in both notations for versions up to and including 8 (e.g., 5 and 1.5, 8 and 1.8). For versions 9 and later, use the single version number (e.g., 9, 10, 11, etc.).`
==>
`Old JDK version numbering is supported (e.g. 1.8 for Java 8) as well as just the major JDK version alone (e.g. 8) is supported.`

Reply:
done